geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/ZIncrByExecutor.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.redis.internal.executor.sortedset;
+
+
+
+import java.util.List;
+
+import org.apache.geode.redis.internal.executor.AbstractExecutor;
+import org.apache.geode.redis.internal.executor.RedisResponse;
+import org.apache.geode.redis.internal.netty.Command;
+import org.apache.geode.redis.internal.netty.ExecutionHandlerContext;
+
+public class ZIncrByExecutor extends AbstractExecutor {
+  @Override
+  public RedisResponse executeCommand(Command command, ExecutionHandlerContext context) {
+    RedisSortedSetCommands redisSortedSetCommands = context.getRedisSortedSetCommands();
+    List<byte[]> commandElements = command.getProcessedCommand();
+
+    byte[] increment = commandElements.get(2);
+    byte[] member = commandElements.get(3);
+
+    byte[] retVal = redisSortedSetCommands.zincrby(command.getKey(), increment, member);
```

Comment:
we return `Infinity` and `-Infinity` because that is the result of converting `POSITIVE_INFINITY` and `NEGATIVE_INFINITY` from double to byte[]. `jedis.zincrby` returns a double, so it will always end up being `Infinity` and `-Infinity` instead of `inf` and `-inf`.
I am also not sure if this is a problem, but it would require returning a byte[] from `jedis.zincrby`, which I don't think is a good idea

Reply:
Discussed this with @DonalEvans, we created a separate ticket for it since the logic was already present in `Coder.java` before this PR: https://issues.apache.org/jira/browse/GEODE-9342